vignettes/synthdid.Rmd

```diff
@@ -7,47 +7,132 @@ vignette: >
   %\VignetteEncoding{UTF-8}
 ---
 
-```{r, echo = FALSE, message = FALSE}
+```{r, echo = FALSE}
 knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
-set.seed(42)
+ragg_png = function(..., res = 192) {
+  ragg::agg_png(..., res = res, units = "in")
+}
+knitr::opts_chunk$set(dev = "ragg_png", fig.ext = "png")
 ```
 
 ```{r setup}
 library(synthdid)
 library(ggplot2)
-library(mvtnorm)
 ```
 
-# Example usage
+# Example use 
+We estimate the effect of California Proposition 99 on cigarette consumption
```

Comment:
This vignette gives an good intro on using the main functions, but I think there is much more room here for you to explain parts of the paper too... Like, you write one sentence " plot the 10 controls with the largest weights 𝜔𝑖", I don't know if the typical user remembers the greek letters from the paper.

It does not hurt to have a recap here in the intro - you can add as much latex math as you want (within reason of course...)